data/campaigns/Under_the_Burning_Suns/scenarios/06a_In_the_Tunnels_of_Trolls.cfg

```diff
@@ -929,7 +941,7 @@
 
         [message]
             speaker=narrator
-            message= _ "Any unit that ends its turn on a lava hex, except those who can fly over the lava, will take 25 damage at the beginning of the next turn. This lava damage can kill units. Flying units will just take $heat_damage damage per turn when flying over lava, though they too can die if they spend too much time over it. Also because of the heat in the cavern, all units on cave floor hexes will take $heat_damage damage at the start of each turn. This heat damage can reduce a unit to 1 hit point, but it can’t kill it."
+            message= _ "Any unit that ends its turn on a lava hex, except those who can fly over the lava, will take $lava_damage damage at the beginning of the next turn. This lava damage can kill units. Flying units will just take $heat_damage damage per turn when flying over lava, though they too can die if they spend too much time over it. Also because of the heat in the cavern, all units on cave floor hexes will take $heat_damage damage at the start of each turn. This heat damage can reduce a unit to 1 hit point, but it can’t kill it."
```

Comment:
Is the "can't kill" right? It contradicts the comment on line 801.

Reply:
both the "can kill" in the comment and the "can't kill" in this string are unchanged from the current revision; I forget which is correct...

Reply:
oh wait a moment, I see what's going on here... there are 2 different sources of damage here: the heat damage, which is the one that can't kill units, and the lava damage, which is the one that *can* kill units... maybe this string should be broken up so it's not talking about both in a single string?

Reply:
I adjusted the string in [dd0c083](https://github.com/wesnoth/wesnoth/pull/8421/commits/dd0c08340433d9d125b45e46e014426d46b0fdc3), but it might still be worth considering splitting it...

Reply:
I think splitting is necessary, otherwise it's a wall of text that says "lava damage" at the start and "can't kill" at the end.

Reply:
Split in [c63001d](https://github.com/wesnoth/wesnoth/pull/8421/commits/c63001d8dcdf3e4022beeef08f98c644d15db1cb).